Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/SharedLogic/FareCalculator.hs

```diff
@@ -87,6 +91,15 @@ mkFareParamsBreakups mkPrice mkBreakupItem fareParams = do
       tollChargesCaption = show Enums.TOLL_CHARGES
       mbTollChargesItem = mkBreakupItem tollChargesCaption . mkPrice <$> fareParams.tollCharges
 
+      insuranceChargeCaption = show Enums.INSURANCE_CHARGES
+      mbInsuranceChargeItem = mkBreakupItem insuranceChargeCaption . mkPrice <$> fareParams.insuranceCharge
+
+      cardChargeOnFareCaption = show Enums.CARD_CHARGE_ON_FARE
```

Comment:
Why card charges needs to be two separate values?

Reply:
Merged into one enum `CARD_CHARGES` and same will be communicated to bap.